app/Helpers/DatabaseHelper.php

```diff
@@ -50,7 +50,7 @@ public static function getDevelopersUrlList($gameid){
         return $res;
     }
 
-    public static function getResourcePathArray($id){
+    public static function getResourcePathArray($id) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->